services/category.js

```diff
@@ -4,7 +4,12 @@ const conditionCreator = require('~/utils/categories/conditionCreator')
 
 const categoryService = {
   getCategories: async (searchFilter, skip, limit) => {
-    return await Category.find(searchFilter).skip(skip).limit(limit).lean().exec()
+    return await Category.find(searchFilter)
+      .skip(skip)
+      .limit(limit)
+      .sort([['totalOffers', -1]])
```

Comment:
I think it would be better to use object in here { totalOffers: -1 } (like in subjects sorting)

Reply:
yeap, it would like nice